pyupgrade/_plugins/six_moves.py

```diff
@@ -0,0 +1,53 @@
+import ast
+import functools
+from typing import Iterable
+from typing import Tuple
+
+from tokenize_rt import Offset
+
+from pyupgrade._ast_helpers import ast_to_offset
+from pyupgrade._data import register
+from pyupgrade._data import State
+from pyupgrade._data import TokenFunc
+from pyupgrade._token_helpers import replace_name
+
+
+NAMES = {
+    'range': 'range',
+    'xrange': 'range',
+}
```

Comment:
since they're currently always `range` -- this can just be inlined as `'range'` right?